smarts/core/agent_interface.py

```diff
@@ -237,7 +243,7 @@ def from_type(requested_type: AgentType, **kwargs):
             max_episode_steps:
                 The total number of steps this interface will observe before expiring
         """
-
+        vehicle_type = kwargs.get("vehicle_type", "car")
```

Comment:
I just realized we don't need to set the vehicle type here at all, this line will take care of overwriting any parameters for us:

https://github.com/huawei-noah/SMARTS/pull/214/files#diff-09e41aaa060eed1eb0690b068f085ab0de6a9d7aae708a50c2cef760936d49f3R326